user-service/controller/user-controller.js

```diff
@@ -128,9 +128,9 @@ export async function userLogout(req, res) {
 
 export async function userAuthentication(req, res) {
     try {
-        const username = req.body.username;
+        const username = req.query.username
+        const token = req.query.auth

```

Comment:
Hmm why is it called "auth" not "token"?

Reply:
> Hmm why is it called "auth" not "token"?

The name of the cookie is called 'auth' instead of token, inherited this name from the cookie.

Reply:
Okay fine.